static/docs/user-guide/data-sharing/mounted-cache.md

```diff
@@ -0,0 +1,144 @@
+# Sharing Data Through a Mounted Cache
```

Comment:
this one replace the shared server use case? I think the explanation in the use case article is better in a sense that it gives the context why a single shared server is used in the first place

Reply:
> this one replace the shared server use case?

Sorry, you got this one wrong. This one is replacing the shared server use case: https://dvc-org-pr-784.herokuapp.com/doc/user-guide/data-sharing/shared-server

Reply:
@dashohoxha then, I'm even more confused since the `shared-server.md` is still part of this PR.